### ISSUE TITLE
It is not possible to get the `referer` header field

### ISSUE THREAD

**Issue by rnowosielski** [awslabs/aws-serverless-java-container#208]
* Framework version: 1.2
* Implementations: Jersey

## Scenario

I am trying to retrieve the header from the original request using the `@HeaderParam("Referer")` annotation e.g
  
    @PUT
    @Path("{url}")
    @Produces("application/json")
    public Response doSomething(@HeaderParam("Referer") String referer) throws Exception {
           logger.info("Referer: " + referer);
    }

## Expected behavior

The logger should log the value of the `referer` header that was passed when calling the endpoint

## Actual behavior
  
The log entry states

    [main] INFO LambdaHandler - Referer: null

## Steps to reproduce

Create an endpoint according to the snippet in the "use case" section, and then call the endpoint by passing `Referer` header on the request.

**Comment by SAPessi** [comment]
Hey @rnowosielski, I just wrote a quick unit test for this. A `Referer` header is indeed not injected by Jersey, even though I can see it in the event. Funnily enough, when I use the exact same method and `@HeaderParam` annotation for an `X-Custom-Referer` header it gets injected correctly. This looks like Jersey's doing. I'm investigating a little bit further.

**Comment by SAPessi** [comment]
Never mind, investigated a bit more and it looks like it was indeed a bug. Committing a fix now.

**Comment by rnowosielski** [comment]
@SAPessi Is there a way to download the most recent snapshot version, so I could test it in my code?

**Comment by SAPessi** [comment]
Yes @rnowosielski, changes are committed to the `core` branch. Just download the code for that branch, `mvn install`, then add `1.3-SNAPSHOT` as a dependency in your code.

**Comment by akrambek** [comment]
@sapessi I am having the same issue and seems to be cause by `https://github.com/awslabs/aws-serverless-java-container/blob/master/aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsProxyHttpServletRequest.java#L808` Any reason why it is replaced with `getIdentity().getCaller()`?!

**Comment by sapessi** [comment]
Hey @akrambek, that's where API Gateway stores the caller/referer. Could you test with `1.3-SNAPSHOT` from the `core` branch to see if that fixes your issue?

**Comment by akrambek** [comment]
hey @sapessi yep it works thanks! Looking forward to the official release.

**Comment by rnowosielski** [comment]
@sapessi do you have some rough estimate when 1.3 might go out ?

**Comment by zsc347** [comment]
> Hey @akrambek, that's where API Gateway stores the caller/referer. Could you test with `1.3-SNAPSHOT` from the `core` branch to see if that fixes your issue?

Seems API Gateway do not store `referer` header to `getIdentity().getCaller()` anymore. But I can get it from `multiValueHeaders`, Could you check  latest behavior?
